### ISSUE TITLE
Package is not working anymore: cannot find module '@cedx/gulp-david'

### ISSUE THREAD

**Issue by cedx** [cedx/gulp-david#11]
It seems that the NPM `require` resolution does not find this module...

```
Error: Cannot find module '@cedx/gulp-david' from '/repo'
    at Function.module.exports [as sync] (/repo/node_modules/resolve/lib/sync.js:33:11)
    at requireFn (/repo/node_modules/gulp-load-plugins/index.js:65:25)
    at Object.get [as smartling] (/repo/node_modules/gulp-load-plugins/index.js:108:41)
    at Object.<anonymous> (/repo/gulpfile.js:122:54)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
```

**Comment by cedx** [comment]
Confirmed: the `lib` folder containing the source code is not packaged into the NPM release, due to the absence of a `.npmignore` file...
The NPM publishing used the `.gitgnore` file... which ignore the `lib` folder 😞

**Comment by cedx** [comment]
Fixed in version 3.0.2.
